runtime/src/rpc.rs

```diff
@@ -69,6 +74,18 @@ impl InterBtcParachain {
         let api: RuntimeApi = ext_client.clone().to_runtime_api();
 
         let runtime_version = ext_client.rpc().runtime_version(None).await?;
+        if let Some(spec_name) = runtime_version.other.get("specName") {
```

Comment:
We should probably also fail if the specName somehow doesn't exist. So I would suggest to use `runtime_version.other.get("specName").unwrap_or_default()` or something similar